### ISSUE TITLE
User creation error

### ISSUE THREAD

**Issue by igrizzli** [dancing-elf/teamcity-telegram-plugin#2]
Hello, i have next issue on user creation: 

```
INFO -   jetbrains.buildServer.SERVER - Unable to create user 'username': notificator_telegram_telegram-chat-id:Chat id should be a number
```

TeamCity version 2017.1

**Comment by dancing-elf** [comment]
Hi!
Looks like "Telegram chat id" field contains Telegram login value. As I remember it should contains only digits. This field should contains chat id.
Set up of plugin consists of two step:
1) create telegram bot
2) get telegram chat id

Telegram bot can't send messages by username or to user who never write to it. So when Teamcity is running send any message to already created bot. Response should contains you chat id.
Complicated scheme but I don't know how to simplify it :(

**Comment by igrizzli** [comment]
I just try to create user, i have no field for telegram chat id on user creation form.

![image](https://cloud.githubusercontent.com/assets/960937/25028834/18912ca6-20c0-11e7-80fd-cb761b7a0089.png)

**Comment by dancing-elf** [comment]
No, it's standard teamcity form. Chat id input field we can see in first teamcity form after login. Let's see screen from README.md again:
![profile_page](https://cloud.githubusercontent.com/assets/1998279/25071067/3964f7c2-22bf-11e7-9abc-7b603051d6c1.png)
Plugin fields we can see in the red rectangle.

**Comment by igrizzli** [comment]
Yes, i know that this is standard form. 

I use it for creating user, but it's not created with next log message
```
INFO -   jetbrains.buildServer.SERVER - Unable to create user 'username': notificator_telegram_telegram-chat-id:Chat id should be a number
```

User creating form has no field for telegram notifier, but you can't create any user until telegram plugin is on

**Comment by dancing-elf** [comment]
Yes, It's looks like a bag. Thanks!

**Comment by dancing-elf** [comment]
I think this problem is resolved in v1.2 (recently published)

**Comment by igrizzli** [comment]
Yes it's solved thx
